Fix faulty default `command` to something that actually works
(refer to https://todo.sr.ht/~kennylevinsen/greetd/21).

Change the `user` to the one we have created for greetd.

--- a/config.toml
+++ b/config.toml
@@ -6,11 +6,11 @@
 # The default session, also known as the greeter.
 [default_session]
 
-# `agreety` is the bundled agetty/login-lookalike. You can replace `$SHELL`
+# `agreety` is the bundled agetty/login-lookalike. You can replace `/bin/sh`
 # with whatever you want started, such as `sway`.
-command = "agreety --cmd $SHELL"
+command = "agreety --cmd /bin/sh"
 
 # The user to run the command as. The privileges this user must have depends
 # on the greeter. A graphical greeter may for example require the user to be
 # in the `video` group.
-user = "greeter"
+user = "greetd"
